src/main/resources/GradleStart.java

```diff
@@ -1,6 +1,4 @@
 import com.google.common.base.Strings;
```

Comment:
I think GradleStart should targets 1.6 instead of 1.8 so Can you revert?

Reply:
Ok, I will change that but why when plugin targets 1.8 we should use 1.6 in this file

Reply:
GradleStart file is compiled on the time mod is built. in the mod template gradle file, source & target version is 1.6 so I think we should keep 1.6 on this file.

Reply:
now uses guava again